analysis/utils.py

```diff
@@ -466,83 +466,7 @@ def which_people_are_companies(people, sentences, corefs):
     return list(companies)
 
 
-def get_people_mentioned(sentences, corefs=None, include_gender=False,
-                         exclude_companies=True):
-    """
-    Process the 'sentences' object returned by CoreNLP's annotation
-    to get a set of people mentioned.
-    It is a list of dictionaries -- one per sentence.
-    If exclude_companies is True, then run our heuristics to
-        get rid of company names.
-    The key we're most concerned about in the dictionary is the tokens one,
-    which contains elements like this for each token in the text.
-    u'tokens': [{u'after': u'',
-                u'before': u'',
-                u'characterOffsetBegin': 0,
-                u'characterOffsetEnd': 6,
-                u'index': 1,
-                u'lemma': u'Google',
-                u'ner': u'ORGANIZATION',
-                u'originalText': u'Google',
-                u'pos': u'NNP',
-                u'speaker': u'PER0',
-                u'word': u'Google'},
-                ...
-                ]
-    """
-
-    people_mentioned = {}
-    honorifics = _get_honorifics(sentences)
```

Comment:
Can we delete the `_get_honorifics()` function as well? It looks like it was only used here.